src/test/java/ru/ayaz/CommentTest.java

```diff
@@ -0,0 +1,49 @@
+package ru.ayaz;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+
+public class CommentTest {
+
+    @Test
+    public void shouldBeEqualWhenReferencesAreSame() {
+        Comment comment = new Comment("Ayaz", "Nice video!");
+        Comment anotherComment = comment;
+
+        assertTrue(comment.equals(anotherComment));
+    }
+
+    @Test
+    public void shouldNotBeEqualWhenInstancesAreDifferent() {
+        Comment comment = new Comment("Ayaz", "Nice video!");
+        Object object = mock(Video.class);
+
+        assertFalse(comment.equals(object));
+    }
+
+    @Test
+    public void shouldNotBeEqualWhenAuthorsAreDifferent() {
```

Comment:
I tried to add such test. It succeed and that means we have no need in adding it. It doesnt force us to write new functionality